### PR TITLE
Car Port: 2021 Camry Hybrid TSS 2.5

### PR DESCRIPTION
**Pull request by alancyau**
Followed the TSS 2.5 port for Highlander and Camry. Hopefully this is everything that's needed. Confirmed with user that OP works.

**Checklist**
- [X] added to README
- [ ] test route added to [test_car_models](../../selfdrive/test/test_car_models.py)
- [ ] route with openpilot:
- [ ] route with stock system:

Little confused about the checklist 2-4. The route added to test_car_models.py was with all OP working.

Dongle ID: ffccc77938ddbc44
Discord user: pjankovi

### CODE REVIEW

File: selfdrive/car/toyota/values.py [hunk 1]
```diff
@@ -286,7 +290,11 @@ class CAR:
 
 # Don't use theses fingerprints for fingerprinting, they are still needed for ECU detection
 IGNORED_FINGERPRINTS = [CAR.RAV4H_TSS2, CAR.HIGHLANDERH_TSS2, CAR.LEXUS_RXH_TSS2, CAR.PRIUS_TSS2,
+<<<<<<< HEAD
+                        CAR.LEXUS_NX, CAR.CAMRY_TSS2, CAR.CAMRYH_TSS2]
+=======
                         CAR.LEXUS_NX, CAR.CAMRY_TSS2, CAR.LEXUS_ESH]
+>>>>>>> upstream/master
```
Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:297]: merge conflict

Comment (adeebshihadeh) [inline selfdrive/car/toyota/values.py:297]:
```suggestion
                        CAR.LEXUS_NX, CAR.CAMRY_TSS2, CAR.CAMRYH_TSS2, CAR.LEXUS_ESH]
```

**Comment by adeebshihadeh** [comment]
You added the test car models route correctly, so that takes care of items 2-3. As for item 4, a stock route is more of a nice to have, especially for Toyota.

**Review by alancyau — COMMENTED** [review]
looks good.

**Comment by adeebshihadeh** [comment]
Nice work, thanks for the PR!

**Pull request merged**